tools/DocGen/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "DocGen": {
       "commandName": "Project",
-      "commandLineArgs": "/path/to/docs --config --dbsize --jsonrpc --metrics"
+      "commandLineArgs": "/path/to/docs --config --jsonrpc --metrics"
```

Comment:
Why are we removing the `--dbsize` flag?

Reply:
We don't. It's for local launch, where it doesn't work without actual data.